test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -2900,9 +2931,7 @@ private void generateSplitsFile(final String splitsFile, final int numItems, fin
   }
 
   private Text getRandomText(final int len) {
-    int desiredLen = len;
-    if (len > 32)
-      desiredLen = 32;
+    var desiredLen = Math.min(len, 32);
```

Comment:
This isn't a great use of `var`. `int` would be better here, since the type isn't obvious from the variable name or its assignment.